lerna.json

```
@@ -1,4 +1,4 @@
 {
   "packages": ["packages/*"],
   "version": "1.0.0"
-}
+}
```

[Formatting] Nit: unrelated change?

```suggestion
}

```